src/svgui/svgui.ts

```diff
@@ -250,6 +250,68 @@ export class Pair extends UIElement {
     }
 }
 
+/*
+ * Image for element.
+ * Backbone properties:
+ *     imageUrl: string
+ *     background: string (color)
+ *     borderColor: string (color)
+ *     padding: {top: number, right: number, bottom: number, left: number}
+ *     minSize: {x:number, y:number}
+ */
+export class Image extends UIElement {
+    public image: d3.Selection<any>;
```

Comment:
`public` is unnecessary here. But should `image` and `imageRect` fields be public in a first place?